certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json

```diff
@@ -2,6 +2,23 @@
   "basePath": "/",
   "consumes": [],
   "definitions": {
+    "Actiontype": {
+      "description": "The Actiontype of the synchronous action that you wish to execute",
+      "title": "",
```

Comment:
@GAC-AJO please fill in this property, empty properties are not allowed.

Reply:
Changed Actiontype to action type.

Reply:
Filled title property